.travis.yml

```diff
@@ -30,12 +30,17 @@ matrix:
   include:
     # Main test suite
     - python: "2.7"
-      env: BOULDER_INTEGRATION=v1 INTEGRATION_TEST=all TOXENV=py27_install
+      env: ACME_SERVER=boulder-v1 PYTEST_ADDOPTS=-n4 TOXENV=integration
```

Comment:
1) I found from testing various values that this setting was the most efficient one when integration tests are run in Travis

2) Because we care less about the execution time during nightly tests, so I let the default behavior, that is to take the number of CPU (so 2), and avoid unnecessarily settings to Travis in this case

3) I think it is doable to set this up in tox.ini, for the integration target env, when `TRAVIS` env is detected

Reply:
1. Yeah? What's the time difference? Are our other tests also faster with PYTEST_ADDOPTS=4?

2. If there's a significant speed increase, I'd personally also like to see it added to the nightly tests. Makes doing tested for things like releases just a little bit faster.

Reply:
1) It is about a 10% less, so 1min or 1min 30s. The gain is marginal for unit tests. It is because in integration tests you have some steps where nothing is done, just waiting (for instance to return a http response, so increasing the parallelization "fills the gaps".

2) OK! I think that if I am able to put that in tox.ini, all integration tests can take advantage of it.

Reply:
2. and 3. done through setenv inheritance (see relevant conversation below) + a `before_script` entry to export conditionally `PYTEST_ADDOPTS="--numprocesses 4"`.